.github/workflows/prow-pr-automerge.yml

```diff
@@ -0,0 +1,17 @@
+# This Github workflow will check every hour for PRs with the lgtm label and will attempt to automatically merge them.
+# If the hold label is present, it will block automatic merging.
+
+name: "Prow merge on lgtm label"
+on:
+  schedule:
+  - cron: "0 * * * *" # every hour
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpmcb/prow-github-actions@v1.1.3
+        with:
+          jobs: 'lgtm'
```

Comment:
Will this check whether the lgtm is given by reviewers in OWNERS file? In other words, can random people lgtm a PR and then it gets automatically merged?

Reply:
Only reviewers in the owner file can give lgtm. https://github.com/jpmcb/prow-github-actions/blob/main/docs/commands.md#owners